modules/troubleshooting-disabling-autoreboot-mco-cli.adoc

```diff
@@ -9,7 +9,7 @@ To avoid unwanted disruptions from changes made by the Machine Config Operator (
 
 [NOTE]
 ====
-Pausing an MCP stops all updates to your {op-system} nodes, including updates to the operating system, security, certificate, as well as any other updates related to the machine config. Pausing should be done for short periods of time only.
+Pausing a machine config pool stops all system reboot processes and all configuration changes from being applied. Pausing an MCP also prevents any automatically rotated certificates from being pushed to the associated nodes, such as the `kube-apiserver-to-kubelet-signer` CA certificate. If the MCP is paused when the `kube-apiserver-to-kubelet-signer` CA certificate is renewed, multiple `oc` commands do not work, including but not limited to `oc debug`, `oc logs`, `oc exec`, and `oc attached`, becasue the node does not have the renewed certificate. Pausing should be done for short periods of time only.
```

Comment:
```suggestion
Pausing a `MachineConfigPool`(MCP) prevents the machine-config-operator from applying any configuration changes on the respective pool. This includes the automatic CA rotation for `kube-apiserver-to-kubelet-signer` CA certificate. If the MCP is paused when the `kube-apiserver-to-kubelet-signer` CA certificate expires and attempts to auto-renew, the new CA certificate is created but not applied across the nodes in the respective pool. This would cause failure in multiple `oc` commands, including but not limited to `oc debug`, `oc logs`, `oc exec`, and `oc attach`. Thus, pausing a MCP should be done with careful consideration over `kube-apiserver-to-kubelet-signer` CA certificate expiry and for short term only.
```